src/layer/Landsat5EOCloudLayer.ts

```diff
@@ -24,4 +25,26 @@ export class Landsat5EOCloudLayer extends AbstractSentinelHubV1OrV2WithCCLayer {
       maxCloudCoverPercent,
     });
   }
+
+  protected getTileLinks(tile: Record<string, any>): Link[] {
+    return [
+      {
+        href: tile.pathFragment,
+        rel: 'self',
+        title: 'EOCloudPath',
+      },
+      {
+        href: `${tile.previewUrl.replace('eocloud', 'creodias')}.JPG`,
+        rel: 'self',
+        title: 'Preview',
+      },
+    ];
+  }
+
```

Comment:
IIUC, this is also not needed, as it is defined on the parent class already? 

(the same below)

Reply:
I'm not sure what you mean with this.

Reply:
Nevermind... :)